cvat/apps/engine/cache.py

```diff
@@ -25,15 +25,24 @@ def __init__(self, dimension=DimensionType.DIM_2D):
     def __del__(self):
         self._cache.close()
 
-    def get_buff_mime(self, chunk_number, quality, db_data):
-        chunk, tag = self._cache.get('{}_{}_{}'.format(db_data.id, chunk_number, quality), tag=True)
+    def get_chunk_mime(self, chunk_number, quality, db_data):
+        cache_key = f'{db_data.id}_{chunk_number}_{quality}'
+        chunk, tag = self._cache.get(cache_key, tag=True)
 
         if not chunk:
-            chunk, tag = self.prepare_chunk_buff(db_data, quality, chunk_number)
-            self.save_chunk(db_data.id, chunk_number, quality, chunk, tag)
+            chunk, tag = self._prepare_chunk_buff(db_data, quality, chunk_number)
+            self._cache.set(cache_key, chunk, tag=tag)
+
         return chunk, tag
 
-    def prepare_chunk_buff(self, db_data, quality, chunk_number):
+    def get_preview(self, key):
```

Comment:
@azhavoro , may be it is a problem. For get_chunk_mime the cache decide how to name the key. For get_preview, it is decided outside.

Should we fix that?